pkg/plugin/query.go

```diff
@@ -88,11 +88,14 @@ func (q *Query) queryInstantURL(expr string, step time.Duration) string {
 	values.Add("step", step.String())
 
 	q.url.RawQuery = values.Encode()
+	if customQueryParams != "" {
+		q.url.RawQuery += "&" + customQueryParams
```

Comment:
I think it is better to use `values.Add("name", "value")` it will add all params to the query

Reply:
updated